compiler/ast/asdl_rs.py

```diff
@@ -649,7 +649,7 @@ def write_ast_def(mod, typeinfo, f):
         #![allow(clippy::derive_partial_eq_without_eq)]
         
         pub use crate::constant::*;
-        pub use crate::Location;
+        pub use ruff_text_size::{TextSize, TextRange};
```

Comment:
My current thinking is that it's highly likely that we make even more changes to `RustPython`to the point where it is no longer be feasible to stream in changes from upstream or we write our own parser from scratch. At that point, we can move the parser into the `ruff` repository and probably rename it to `ruff_python_parser`. With this longterm naming in mind, I prefer keeping `ruff_text_size` to avoid renaming the crate forth and back. 

There's another consideration. Naming it `rustpython-textsize` gives the impression that it is a crate authored by RustPython, which isn't the case.